### PR TITLE
Make and install a pkg-config file on Windows

### LINKED ISSUE #32: @conda-forge-admin, please rerender
### Additional comment:

(need a branch to work on)

### PR DESCRIPTION
**Pull request by conda-forge-linter**
Adapted from
https://github.com/conda-forge/libxml2-feedstock/blob/a4e77d2a8486ec016cde621446ef820b5e0718e2/recipe/0002-Make-and-install-a-pkg-config-file-on-Windows.patch

Here's a checklist to do before merging.
- [x] Bump the build number if needed.

Fixes #32

### CODE REVIEW

**Comment by hmaarrfk** [comment]
Thank you for tackling this. I know that it is "churn" but I do think that keeping things in line with the include structure on unix, is a positive, as a whole.

**Comment by mbargull** [comment]
> Thank you for tackling this. I know that it is "churn" but I do think that keeping things in line with the include structure on unix, is a positive, as a whole.

Yeah, dealing with Windows builds isn't my favorite pastime :D.
But if I wouldn't agree with your change, I hadn't tackled this ;).
I believe it's beneficial to reduce the special-casing on Windows earlier so we don't have to deal with it multiple times later.

**Comment by mbargull** [comment]
@conda-forge/libxslt, I'm merging this as-is to get the ball rolling.
But if you see the need to alter/improve anything, please let me know!

File: recipe/bld.bat [hunk 1]
```diff
@@ -4,8 +4,16 @@ if "%VS_MAJOR%" == "9" (
     set MANIFEST=yes
 )
 
+:: Get libxml2 include paths via pkg-config
+:: N.B.: We may at some point want to try another build system that supports
+::       pkg-config out of the box.
+set "PKG_CONFIG_PATH=%LIBRARY_LIB%\pkgconfig;%LIBRARY_PREFIX%\share\pkgconfig"
```
Comment (hmaarrfk) [inline recipe/bld.bat:10]: They seem to have a CMake based build process too. It might be worthwhile checking it out later.

Comment (mbargull) [reply recipe/bld.bat:10]: Yep, Autotools and CMake; that's what that comment is to hint at ;).

**Pull request merged**